### DISCUSSION TITLE
multiple S3 deployments on a site

### DISCUSSION THREAD

**Discussion by jhatmaker** [WP2Static/wp2static-addon-s3#48] [General]
Is there a way to have multiple S3 deployments on a site? 

A use case is that we have a 2 different S3 buckets, one that feeds a production cloudfront distribution, the other is one that would be used for development/hotfix/customer approval site prior to pushing it to the live static site. 
We can/have used the live WP site, but this will may not accurately portray the production functionality (like if a web developer used a server side plugin).  
Right now we manually have to update all the S3/cloudfront configurations and redeploy.  It would be nice to just disable the production/live deployment when we want to view a static render of the new site, then reactivate both deployments to keep both sites in sync.

**Comment by john-shaffer** [comment]
We definitely want this eventually, but it's not supported right now.

**Reply by jhatmaker** [reply]
Ok Sounds good. Thank you!
